.github/workflows/test.yml

```diff
@@ -0,0 +1,17 @@
+name: "Test"
+
+on:
+  pull_request:
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v2
+      with:
+        go-version: 1.17.1
+    - name: Install libpcap-dev
+      run: sudo apt-get install -y libpcap-dev
```

Comment:
Should we add some documentation somewhere about requiring this for local dev environment?

Reply:
I added it to the README.